robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowViewTreeObserver.java

```diff
@@ -14,6 +14,7 @@
 @Implements(ViewTreeObserver.class)
 public class ShadowViewTreeObserver {
```

Comment:
I think I would prefer to remove this whole shadow rather than reimplementing Android behaviour. You can fire the listeners yourselfusing dispatchOnPreDraw() and dispatchOnGlobalLayout().

I'm not sure if there is really a need to get hold of the listeners themselves, but if there is we could just make a non-android accessor that gets the field by reflection.

Thoughts?